stable/traefik/README.md

```diff
@@ -87,6 +87,9 @@ The following tables lists the configurable parameters of the Traefik chart and
 | `acme.persistence.size`         | Minimum size of the volume requested                                 | `1Gi`                                     |
 | `dashboard.enabled`             | Whether to enable the Traefik dashboard                              | `false`                                   |
 | `dashboard.domain`              | Domain for the Traefik dashboard                                     | `traefik.example.com`                     |
+| `dashboard.address`             | Address for the Traefik dashboard service                            | `:8080`                                   |
+| `dashboard.public`              | Whether the Traefik dashboard should be exposed via an Ingress       | `true`                                    |
+| `dashboard.http_auth`           | List of user/password combinations to be used via HTTP Auth          | none                                      |
```

Comment:
Can we stick with camel case to be consistent with the other properties?
-> `dashboard.httpAuth`